org.evuz.visualReader.appdata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.evuz.visualReader.desktop</id>
+    <name>Visual Comic Reader</name>
+    <developer_name>evuz</developer_name>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>ISC</project_license>
+    <summary>Read comics in differents platforms but the same way</summary>\
+</component>
```

Comment:
Could use more content like screenshots, release info, etc. And ideally see if upstream would be interested in it.